### DISCUSSION TITLE
Artisan command `autocomplete:publish` doesn't exists

### DISCUSSION THREAD

**Discussion by bilogic** [joshhanley/livewire-autocomplete#4] [Bugs] [answered]
@joshhanley jfyi,

`php artisan autocomplete:publish --components` fails
I can't find it in the source either :)

Thank you.

**Comment by joshhanley** [comment, marked as answer]
@bilogic thanks! 😅

I was writing the readme by wishful thinking haha, hence haven't actually implemented it.

Have updated the readme to use `vendor:publish`.

You can use `php artisan vendor:publish --provider="LivewireAutocomplete\LivewireAutocompleteServiceProvider" --tag="autocomplete-components"` to publish the components.

Hope this helps!
